index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>My portfolio</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="./assets/libraries/fontawesome/css/all.css" rel="stylesheet">
+        <link rel="stylesheet" href="./assets/css/style.css">
+    </head>
+    <body>
+        <div class="container">
+            <header class="header">
+                <div class="logo">
+                    My Logo
+                </div>
+                <div class="menu">
+                    <i class="fas fa-bars"></i>
+                </div>
+            </header>
+            <div class="main-content">
```

Comment:
- [ ] I would recommend using the `<section>` tag instead of the `<div>` for better semantics. Remember that for good practices we try to avoid `<div>`s as much as possible :raised_hands:
- [ ] You have done a good job with the background :+1: but since you have applied `padding: 15px` in the `container`, this section leaves a blank space on each side. Please, try to follow the Figma design.

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/138343162-dcd25608-0057-472a-8c8a-3d528fc789e8.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/138343251-b203fc58-3246-4556-aedc-a9a329e9001a.png)